src/pages/test-report/cat-be/test-report.cat-be.page.html

```diff
@@ -103,7 +103,7 @@
         </ion-row>
         <ion-row>
           <ion-col>
-           <uncouple-recouple></uncouple-recouple>
+           <uncouple-recouple [category]="'B+E'"></uncouple-recouple>
```

Comment:
would be nice to get category from state

Reply:
fixed